certdb/testdb/testdb.go

```diff
@@ -0,0 +1,63 @@
+package testdb
+
+import (
+	"database/sql"
+
+	_ "github.com/lib/pq"           // register postgresql driver
+	_ "github.com/mattn/go-sqlite3" // register sqlite3 driver
+)
+
+const (
+	pgTruncateTables = `
```

Comment:
It makes sure all tables in the database are empty. It is a snippet I shamelessly copy/pasted from my colleague. So it could be a overkill, and be simplified further.